src/rippled/lib/txSummary/index.js

```diff
@@ -1,16 +1,8 @@
 import { transactionTypes } from '../../../containers/shared/components/Transaction'
 
-const summarize = {}
-
 const getInstructions = (tx, meta) => {
   const type = tx.TransactionType
-  // Locate the transaction parser which returns "instructions" to be used in transaction specific components
-  // TODO: Remove summarize[type] lookup once all transactions have been moved to the new definition style
-  const mappingFn = transactionTypes[type]?.parser
-    ? transactionTypes[type]?.parser
-    : summarize[type]
-
-  return mappingFn ? mappingFn(tx, meta) : {}
+  return transactionTypes[type]?.parser(tx, meta) || {}
```

Comment:
Thoughts on something like this for a better default case if there's a parser error?
```suggestion
  try {
    return transactionTypes[type]?.parser(tx, meta)
  catch {
    return tx
  }
```

Or should that be handled in a separate PR? I guess the parsers would all need to change as well.

Reply:
Going with a separate PR.  I will create a ticket for this.